lib/util/transformer/length-based-splitter.js

```
@@ -66,6 +71,19 @@ class LengthBasedSplitter extends Stream.Transform {
     this._frameBufferIndex = 0;
   }
 
+  _pushBack (start, end, data) {
+    if (start === end) {
```

if start >= end ?

---

If start is bigger than end, we have serious problems

---

perhaps it makes sense to throw an exception in such case?

---

I don't think you can throw an error in these functions. It will just be swallowed in the `Tranform` machinery.

---

perhaps, writing a log record would be enough then